docs/arch/adr-001.md

```diff
@@ -0,0 +1,59 @@
+# Architectural Decision Record (ADR) 1: Unification of METS Creation in Archivematica
+
+## Context
+
+### Technological forces
+
+- METS Reader & Writer (henceforth [metsrw](https://github.com/artefactual-labs/mets-reader-writer) already exists, is being used in Archivematica, and seems like a good tool for abstracting METS file creation.
+
+- Currently, [Archivematica](https://github.com/artefactual/archivematica) (AM) and the [Archivematica Storage Service](https://github.com/artefactual/archivematica-storage-service) (SS) both create or modify METS files (pointer files are METS files too), using mets-rw or various lxml APIs. This inconsistency makes it difficult to reliably create predictable METS files.
+
+- Artefactual has created a [METS validator](https://github.com/artefactual/mets-validator) project, which contains a Schematron file that validates AM-generated AIP METS files.
+
+- metsrw (v. 0.1.1) cannot create pointer files and it does not support METS file validation, either via XMLSchema or Schematron.
+
+- metsrw (v. 0.2.0) does have support for pointer file creation and for METS file validation generally, both via XMLSchema and Schematron.
+
+- Changing how we write METS files may inadvertently affect the METS files produced; METS validation and related testing will need to prevent this, or formally recognize it if it is intentional.
+
+- At present in AM and SS, METS/PREMIS data structures are encoded at various points as XML strings, MySQL/SQLite database rows, and Python objects (i.e., metsrw or lxml instances).
```

Comment:
I wasn't clear from the decision whether this problem is solved going forward? If I'm a developer in future do I just need to know that as long as I use metsrw I won't be making this problem worse? (I'm assuming this inconsistency is a problem)